pkg/asset/installconfig/installconfig_test.go

```diff
@@ -189,7 +189,21 @@ platform:
     region: us-east-1
 pullSecret: "{\"auths\":{\"example.com\":{\"auth\":\"authorization value\"}}}"
 network:
```

Comment:
Use a field name that is **clearly** unknown rather than a field name that is similar to a known field. It is hard reading the test case input to spot immediately which field is wrong.